js/forum/src/components/EditUserModal.js

```diff
@@ -33,26 +34,55 @@ export default class EditUserModal extends Modal {
   }
 
   content() {
+    const topObj = this;
```

Comment:
Well since you aren't inside of the actual class of the button, you can use `this`.